monitoring/scripts/comprehensive_test.py

```
@@ -0,0 +1,752 @@
+            # Check target health
+            response = requests.get('http://localhost:9090/api/v1/targets', timeout=10)
+            if response.status_code == 200:
+                data = response.json()
+                targets = data.get('data', {}).get('activeTargets', [])
+                
+                healthy_targets = [t for t in targets if t.get('health') == 'up']
+                if len(healthy_targets) > 0:
+                    results['target_health'] = True
+                    logger.info(f"✅ {len(healthy_targets)}/{len(targets)} targets healthy")
+                else:
+                    logger.warning(f"⚠️  No healthy targets found")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)